napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -318,9 +356,11 @@ def _restore_shortcuts(self, row):
     def _mark_conflicts(self, new_shortcut, row) -> bool:
         # Go through all layer actions to determine if the new shortcut is already here.
         current_action = self._table.item(row, self._action_col).text()
-        actions_all = self._get_layer_actions()
+        actions_all = self._get_potential_conflicting_actions()
         current_item = self._table.currentItem()
-        for row1, (action_name, action) in enumerate(actions_all.items()):
+        for conflicting_row, (group, (action_name, action)) in enumerate(
+            actions_all
+        ):
```

Comment:
Here, we iterate over all actions, even if we create a conflict mark only for actions from the current view. 
So to understand what happens here you need to read multiple functions. 
We're also detecting only first conflict.

Reply:
You are totally right! Changing the conflict message to support showing info in a list manner could make things look something like this:

* Multiple conflicts detected:
![imagen](https://github.com/user-attachments/assets/a49909a3-6950-43d5-b06a-5b24a9f6b4f8)

* Single conflict detected:
![imagen](https://github.com/user-attachments/assets/1ec5088d-3d93-48a3-82eb-17737f596456)

Making the message look like this makes sense or maybe other approach to show the multiple conflicts info should be explored?

Also, thanks for giving this a review @Czaki !

Reply:
Last commit ([e9d52cd](https://github.com/napari/napari/pull/7231/commits/e9d52cdee00a20a2824e46e6cea2513ccaadcf8e)) adds the changes required to support showing multiple conflicts as shown over the screenshot above. Let me know if things working/looking that way make sense or if something needs to be changed!